.appveyor.yml

```diff
@@ -1,3 +1,5 @@
+version: "{branch} #{build}
```

Comment:
I'm not familiar with the meaning of this line.
What is the benefit of adding this?

Reply:
And yea, same here. I wouldn't allow broken stage. I think amending or squashing is the better approach.

Reply:
Agree, I will force push from home. I try commit to pr from phone.

Reply:
By default, only the build number is shown, so it is more convenient to look at which branch is build.